src/migrate/migrate-sample-data.ts

```diff
@@ -432,72 +438,71 @@ export const testDataV1 = {
 };
 
 export const testDataV2 = {
-  'c/5vm3movfjjpjs5sanpmnd4n965kvn04p/d': {
-    indexes: [],
-    meta: {
-      basisHash: 'hjera9r19os7oumi22s4mkba84jdn16e',
-      mutationID: 1,
-      mutatorArgsJSON: [1],
-      mutatorName: 'mutator_name_1',
-      originalHash: null,
-      type: 2,
-    },
-    valueHash: 'bk5r3f2ui3neg8gh8na8pn8o1ij0n88p',
-  },
-  'c/5vm3movfjjpjs5sanpmnd4n965kvn04p/m': [
-    'bk5r3f2ui3neg8gh8na8pn8o1ij0n88p',
-    'hjera9r19os7oumi22s4mkba84jdn16e',
-  ],
-  'c/5vm3movfjjpjs5sanpmnd4n965kvn04p/r': 1,
-  'c/bk5r3f2ui3neg8gh8na8pn8o1ij0n88p/d': [['local', '1']],
-  'c/bk5r3f2ui3neg8gh8na8pn8o1ij0n88p/r': 1,
-  'c/c50qh3sv6pv6g956205g5566ftjd20pr/d': {
+  'c/00000000000000000000000000000000/r': 2,
+  'c/3qamjl84no3b1rsdikdh3leu3vraihlc/d': {
     indexes: [],
     meta: {
-      basisHash: 'hjera9r19os7oumi22s4mkba84jdn16e',
-      cookieJSON: 'sync_cookie_2',
+      basisHash: null,
+      cookieJSON: null,
       lastMutationID: 0,
       type: 3,
     },
-    valueHash: 'jdh7scesonbpmik7k00vnb4ska0goree',
+    valueHash: '00000000000000000000000000000000',
```

Comment:
Empty hash here is ugly and requires special casing in other places of the system.